src/Methods/PaymentAbstract.php

```diff
@@ -239,4 +242,13 @@ public function isSwitchableTo($orderId = null): bool
 
         return false;
     }
+    /**
+     * Check if it is allowed to switch from this payment method
+     * @return bool
+     */
+    public function isSwitchableFrom($orderId = null): bool
+    {
+        // TODO ADD AMAZON PAY
```

Comment:
Here is still a TODO open.
And we are really sure that it is allowed to switch away from all PayOne payment methods?

Reply:
I have added isSwichableTo as false for AmazonPay, in order for AmazonPay to be disabled in the reinit Payment List.
Amazon Pay is not implemented yet into the reinit - this was agreed because of the AmazonLogin button that cannot be added to the reinit Container - i guess this should be analysed, but for the moment this method will be left out.